robottelo/cli/import_.py

```diff
@@ -36,13 +36,49 @@
     user                          Import Users (from spacewalk-report users).
 
 """
-from robottelo import ssh
+import os
+from robottelo import manifests, ssh
 from robottelo.cli.base import Base
 
 
 class Import(Base):
     """Imports configurations from another Satellite instances."""
     command_base = 'import'
+    repos = {
```

Comment:
@elyezer:
yes, i am aware of the repos defined in `constants.py`. But these are of different format. Hammer import transitions them.